cartography/intel/aws/ec2/vpc.py

```diff
@@ -32,11 +32,11 @@ def _get_cidr_association_statement(block_type: str) -> str:
         new_block.cidr_block = block_data.$block_cidr,
         new_block.block_state = block_data.$state_name.State,
         new_block.block_state_message = block_data.$state_name.StatusMessage,
-        new_block.lastupdated = {aws_update_tag}
+        new_block.lastupdated = {update_tag}
```

Comment:
Ooh good catch; I wonder if there are other places where this needs to be  updated.

Reply:
Is the goal to get rid of all instances of `aws_update_tag` for just `update_tag`? Perhaps just a blank grep | sed should do.

Reply:
Yeah, that probably would be fine. This is a bit of a special case in that the "`load` string" is separate from the `load()` function so when I did #530 I missed it. Thanks for catching it.